### PR TITLE
LLNL guest post

### PR DESCRIPTION
**Pull request by Dahianna**
[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/llnl-post)

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/18f/18f.gsa.gov/llnl-post/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/llnl-post/README.md)

Changes proposed in this pull request:
- Add author Ian Lee
- Publish LLN Lab post
- Add LLN Lab repo images

Checklist:
- [x] All images being added have been optimized **_--> [learn more](18f.gsa.gov/styleguide/images/#using-and-optimizing-jpg-and-png) about how to optimize images <--_**

/cc @awfrancisco

### CODE REVIEW

File: _posts/2018-07-12-the-case-for-open-source-software.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+One of our most popular open source HPC projects is ZFS on Linux, which,
+as the name states, is a version of the ZFS file system that supports
+the Linux operating system. Embodying the community’s quid pro quo, we
+could adapt the software to Linux because the [original
+ZFS](http://open-zfs.org/wiki/History) (which only supported the
+Solaris operating system) was open source.
+
+We create software that resonates with the open source community due to
+its flexibility with a variety of use cases. LNLL’s Spack, a collection
```
Comment (IanLee1521) [inline _posts/2018-07-12-the-case-for-open-source-software.md]: Minor typo: `LNLL` -> `LLNL`

File: _posts/2018-07-12-the-case-for-open-source-software.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  solution. OSS is not for every organization, but for an organization
+  like LLNL, whose job is to build innovative solutions for national
+  security, OSS has been an essential tool.
+    <span>- Terri Quinn, LLNL’s Advanced Simulation and Computing Program leader</span>
+</div>
+
+For example, we’re developing open source tools with the NNSA’s
+[Advanced Simulation and Computing Program](https://www.exascaleproject.org/more-on-the-software-that-underpins-the-exascale-computing-project/) to make HPC codes portable to next-generation architectures. The DOE’s [Exascale Computing Project](https://www.exascaleproject.org/) depends on open source technology to share work among participating institutions, including LLNL and other national labs.
+
+One of our most popular open source HPC projects is ZFS on Linux, which,
```
Comment (IanLee1521) [inline _posts/2018-07-12-the-case-for-open-source-software.md]: Can we add the link "http://zfsonlinux.org" to the text "ZFS on Linux" ?

File: _posts/2018-07-12-the-case-for-open-source-software.md (hunk 1 above)
Comment (IanLee1521) [reply _posts/2018-07-12-the-case-for-open-source-software.md]: Also, can we add a link of `[LLNL's Spack](https://spack.io)`

Comment (Dahianna) [reply _posts/2018-07-12-the-case-for-open-source-software.md]: Typo fixed and link added

File: _posts/2018-07-12-the-case-for-open-source-software.md (hunk 2 above)
Comment (Dahianna) [reply _posts/2018-07-12-the-case-for-open-source-software.md]: Added the link

**Comment by hauten** [comment]
Good morning, 18F team. Would it be possible to move the lead image so it shows the top of the software.llnl.gov home page with the Lab's name and website title? More like: 
portal home

**Comment by IanLee1521** [comment]
Side note, @Dahianna -- I think the preview link in the top post here needs to be: https://federalist-proxy.app.cloud.gov/preview/18f/18f.gsa.gov/llnl-post/2018/07/12/the-case-for-open-source-software/

**Comment by awfrancisco** [comment]
Thanks. We're working on fixing the header image and then we'll get this published.

**Pull request merged**